new-lamassu-admin/src/App.js

```diff
@@ -119,19 +122,47 @@ const Main = () => {
 
 const App = () => {
   const [wizardTested, setWizardTested] = useState(false)
+  const [userData, setUserData] = useState(null)
+  const [loading, setLoading] = useState(true)
+
+  const url =
+    process.env.NODE_ENV === 'development' ? 'https://localhost:8070' : ''
+
+  useEffect(() => {
+    getUserData()
+  }, [])
```

Comment:
check if you don't get a console warning for not adding getUserData to the array of dependencies of useEffect
If so, and if after adding you get an infinite loop, you also might need to wrap getUserData on a useCallback hook
You can also move the code to inside of useEffect but I'm not 100% sure it won't cause a different set of problems

https://stackoverflow.com/a/60836660 check this answer for useCallback example

Reply:
It does show a warning. My solution was just to put the axios request inside the useEffect, instead of isolating the request in a function and calling it inside the useEffect.